enhancements/windows-containers/windows-node-egress-proxy.md

```diff
@@ -167,27 +168,33 @@ OLM is a subscriber to these `Proxy` settings -- it forwards the settings to the
 container will automatically get the required `NO_PROXY`, `HTTP_PROXY`, and `HTTPS_PROXY` environment variables on startup.
 In fact, OLM will update and restart the operator pod with proper environment variables if the `Proxy` resource changes.
 
-On the Linux side this is achieved through `MachineConfigs` -- when global proxy settings are changed, the ignition
-config is re-rendered by MCO (for whom proxy info is injected by CVO) to include a systemd unit
-`EnvironmentFile=/etc/mco/proxy.env`, with proxy vars set in this file.
-
-WMCO will retrieve the proxy variables by watching the `rendered-worker` `MachineConfig` for changes and parse info from
-the `proxy.env` file. This could be an expensive operation, but since we already parse ignition from the `MachineConfig`
-as part of node bootstrap, the technical cost and complexity is reduced.
-
-Then, once it has the required values, the operator will set the 3 proxy environment variables on the Windows instance
-during node configuration/reconciling through Powershell.
-```powershell
-[Environment]::SetEnvironmentVariable('<name>', '<value>', 'Machine')
+WMCO the store the 3 proxy variables and their values in the `windows-services` ConfigMap.
+This will be done by adding a `EnvironmentVars` key to the services ConfigMap spec.
+The new key will be a list of name-value pairs representing environment variables to be set on Windows instances:
+```json
+[
+  {
+    "name": "name of the environment variable",
+    "value": "value the variable should be set to on the instance"
+  }
+]
 ```
-Note that we will set these variables at the `Machine` scope, the highest level, to ensure all WMCO-managed services
-(kubelet, containerd, etc.) will inherit them. Since Windows services read environment vars from the system registry,
-and the registry does not update until a shutdown/startup cycle, this will require rebooting the instance.
+
+WICD, which uses the ConfigMap data to configure Windows services, will now have the added responsibility to ensure the
+proper setting of these `EnvironmentVars` variables on the instance. 
```

Comment:
setting the variables on the instance -> ensuring the variables are correctly set for the service